v2/manager.go

```diff
@@ -427,7 +433,13 @@ func (c *Manager) Stat() (*stats.Metrics, error) {
 		SwapUsage:             getStatFileContentUint64(filepath.Join(c.path, "memory.swap.current")),
 		SwapLimit:             getStatFileContentUint64(filepath.Join(c.path, "memory.swap.max")),
 	}
-
+	metrics.MemoryEvents = &stats.MemoryEvents{
```

Comment:
if map `memoryEvents` is empty, do we just let `metrics.MemoryEvents` nil here?

Reply:
updated